docs/_build/html/_static/documentation_options.js

```diff
@@ -0,0 +1,11 @@
+var DOCUMENTATION_OPTIONS = {
```

Comment:
Is any of this stuff in _build generated by the build? If it is it should be gitignore'd